src/layers/XRLayer/XRLayer.js

```diff
@@ -54,12 +54,33 @@ const defaultProps = {
 };
 
 /**
- * This layer serves as the workhorse of the project, handling all the rendering.  Much of it is
- * adapted from BitmapLayer in DeckGL.
- * XR = eXtended Range i.e more than the standard 8-bit RGBA data format
- * (16/32 bit floats/ints/uints with more than 3/4 channels).
+ * @typedef LayerProps
+ * @type {object}
+ * @property {Array.<Array.<number>>} sliderValues List of [begin, end] values to control each channel's ramp function.
+ * @property {Array.<Array.<number>>} colorValues List of [r, g, b] values for each channel.
+ * @property {Array.<Array.<boolean>>} channelIsOn List of boolean values for each channel for whether or not it is visible.
+ * @property {number=} opacity Opacity of the layer.
+ * @property {string=} colormap String indicating a colormap (default: '').  The full list of options is here: https://github.com/glslify/glsl-colormap#glsl-colormap
+ * @property {Array.<number>=} domain Override for the possible max/min values (i.e something different than 65535 for uint16/'<u2').
+ * @property {String=} id Unique identifier for this layer.
+ * @property {function=} onHover Hook function from deck.gl to handle hover objects.
+ * @property {boolean=} isLensOn Whether or not to use the lens.
+ * @property {number=} lensSelection Numeric index of the channel to be focused on by the lens.
+ * @property {number=} lensRadius Pixel radius of the lens (default: 100).
+ * @property {Array.<number>=} lensBorderColor RGB color of the border of the lens (default [255, 255, 255]).
+ * @property {number=} lensBorderRadius Percentage of the radius of the lens for a border (default 0.02).
+ * @property {number} dtype Dtype for the layer.
+ * @property {function=} onClick Hook function from deck.gl to handle clicked-on objects.
+ * @property {Object} modelMatrix Math.gl Matrix4 object containing an affine transformation to be applied to the image.
```

Comment:
Listed as a required fielded; don't see where this gets used. Could be missing something.

Reply:
No I think the push was rejected.  My internet has been flaking in and out.  Switched to hotspot, will try again.